docs/create_benchmarks_table.py

```diff
@@ -24,15 +24,20 @@ def benchmark_to_markdown_row(b: mteb.Benchmark) -> str:
     langs = ",".join(list(agg_langs))
     domains = "[" + ", ".join(agg_domains) + "]" if agg_domains else ""
 
-    task_types = dict(Counter([t.metadata.type for t in b.tasks]))
+    task_types = ", ".join(
+        [
+            f"{name}: {val}"
+            for name, val in Counter([t.metadata.type for t in b.tasks]).items()
+        ]
+    )
 
-    return f"| {name_w_reference} | {n_tasks} | {task_types} | {domains} | {langs} |"
+    return f"| {name_w_reference} | {b.display_name if b.display_name else b.name} | {n_tasks} | {task_types} | {domains} | {langs} |"
```

Comment:
Maybe let's stick with `b.name` to be consistent? Users will have to use `b.name` in the code anyway.

Reply:
Oh I see what you mean now. The mapping is actually very helpful. Let's keep these as is.